projects/TransFusion/configs/transfusion_nusc_pillar_L.py

```diff
@@ -0,0 +1,389 @@
+_base_ = ['mmdet3d::_base_/default_runtime.py']
```

Comment:
```suggestion
_base_ = ['../../../configs/_base_/default_runtime.py']
```
Better to use a relative path to avoid potential problems with openmim.